deepinv/loss/score.py

```diff
@@ -1,41 +1,161 @@
 import torch
-import numpy as np
+import deepinv.physics
 from deepinv.loss.loss import Loss
 
 
 class ScoreLoss(Loss):
     r"""
-    Approximates score of a distribution.
+    Learns score of noise distribution.
 
-    It approximates the score of the measurement distribution :math:`S(y)\approx \nabla \log p(y)`
+    Approximates the score of the measurement distribution :math:`S(y)\approx \nabla \log p(y)`
```

Comment:
isn't it p_\sigma rather than p ?

Reply:
yes, but they anneal sigma to zero during training.